src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/TextEncoder.cs

```diff
@@ -503,12 +503,8 @@ internal unsafe virtual OperationStatus EncodeUtf8(ReadOnlySpan<byte> utf8Source
         /// <summary>
         /// Shim function which can call virtual method <see cref="EncodeUtf8"/> using fast dispatch.
         /// </summary>
-        [MethodImpl(MethodImplOptions.AggressiveOptimization)]
```

Comment:
That's strange. Why would removing this fix the netfx build failure? This commit looks to only be modifying comments and this attribute.

Reply:
That enum value isn't available in the downlevel frameworks. :(